subsys/bluetooth/mesh/pb_adv.c

```diff
@@ -317,6 +351,180 @@ static void protocol_timeout(struct k_work *work)
 	link.rx.seg = 0U;
 	prov_link_close(PROV_BEARER_LINK_STATUS_TIMEOUT);
 }
+
+static struct delayable_adv_ctx *delayable_adv_ctx_alloc(struct bt_mesh_adv *adv,
+							 const struct bt_mesh_send_cb *cb,
+							 void *cb_data, bool retransmit)
+{
+	struct delayable_adv_ctx *ctx;
+	uint16_t random_delay;
+	uint32_t curr_time;
+
+	if (k_mem_slab_alloc(&delayable_advs_buf_pool, (void **)&ctx, K_NO_WAIT)) {
```

Comment:
If is not able to allocate it will return false and it will try to push out a message. And then try to allocate again. So it will not use ctx if it is not set.